synapse/storage/data_stores/main/registration.py

```diff
@@ -281,7 +281,7 @@ def is_server_admin(self, user):
             user (UserID): user ID of the user to test
 
         Returns (bool):
-            true iff the user is a server admin, false otherwise.
+            true if the user is a server admin, false otherwise.
```

Comment:
fyi, "iff" means "if and only if". I think it's clear either way, so no need to change it back.